results/cmd/tests/main.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
I agree with Vincent - we should definitely convert this into a full Go test so we can validate the behavior of the API server. This can likely be largely converted as is for ease of testing.

For local testing, you will probably find [grpc_cli](https://github.com/grpc/grpc/blob/master/doc/command_line_tool.md) to be very useful here.

Reply:
I have created a test file inside same directory as main.go, this tests folder would retire.